packages/nimbuild-corejs/lib/tests/__snapshots__/index.test.js.snap

```diff
@@ -123,7 +123,128 @@ Array [
 
 exports[`index.js creates ie11 corejs minified polyflls 2`] = `90315`;
 
-exports[`index.js handles UA not defined in browserlist 1`] = `undefined`;
+exports[`index.js handles UA not defined in browserlist 1`] = `
```

Comment:
This was a correction.  Snapshot was incorrectly generated due to outdated reference in unit test assertion.